torch/_refs/__init__.py

```diff
@@ -1138,6 +1138,34 @@ def clamp(
     raise ValueError(msg)
 
 
+@out_wrapper
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self", "min"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def clamp_min(
```

Comment:
Should these use elementwise binary helper?

https://github.com/pytorch/pytorch/blob/399b3dcac64917b6d49f1f7e7ba47fa29165b4b0/torch/_refs/__init__.py#L684


I think it would take care of `out` and type promotion and `_maybe_broadcast`?